app/helpers/application_helper.rb

```diff
@@ -5,12 +5,6 @@ def flash_notice(flash, &block)
     content_tag(:div, class: flash_class, &block)
   end
 
-  def fix_url(url)
-    unless url.include?("http")
-      "http://" + url
-    end
-  end
-
   def spacer
```

Comment:
Hah, I don't know.  I think its just a legacy of when I was learning how to use helper methods for the first time.  I'll get rid of it.